Documentation/DataTypes/Properties/GetText.rst.txt

```diff
@@ -400,7 +401,7 @@ levelfield
    field. In order that you can use this function, your fieldname 'user_myExtField' needs be included in the comma
    separated list of ['addRootLineFields']::
 
-      foo = levelfield : -1, user_myExtField, slide
+      foo.data = levelfield : -1, user_myExtField, slide
```

Comment:
I'm not sure about adding the `.data`. It's nowhere and I guess it should not be added as one could use `insertData` and further syntax. This is about the datatype itself, not how to use it in certain circumstances.

What do you think @pi-phi?